programs/merkle-distributor/src/instructions/new_claim.rs

```diff
@@ -176,7 +182,9 @@ pub fn handle_new_claim(
     );
```

Comment:
https://github.com/drift-labs/distributor/pull/1/files#diff-a20f837a789958ff621bf82a85899540c6e5c45fbaeefa74db8d18b2b9cd7d3aR176-R181

might be good to update the `msg!` here too